test/types/create.test.ts

```diff
@@ -40,6 +40,15 @@ Test.create([{ name: 'test' }], { validateBeforeSave: true }).then(docs => {
   expectType<string>(docs[0].name);
 });
 
+Test.create({}).then(doc => {
+  expectType<string>(doc.name);
+});
+
+Test.create([{}]).then(docs => {
+  expectType<string>(docs[0].name);
+});
+
+expectError(Test.create<ITest>({}));
```

Comment:
maybe consider also adding a test for the correct / successful case

Reply:
Fixed :+1: